simphony/testing/utils.py

```diff
@@ -9,7 +9,43 @@
 from simphony.core.data_container import DataContainer
 from simphony.core.cuba import CUBA
 from simphony.cuds.particles import Particle, Bond
-from simphony.cuds.mesh import Point
+from simphony.cuds.mesh import Point, Edge, Face, Cell
+
+
+def compare_particles_datasets(particles, reference, msg=None, testcase=None):
```

Comment:
The `msg` parameter is unused in this method (and in subsequent methods).  I would recommend deleting it.